helm_tests/other/test_git_sync_scheduler.py

```diff
@@ -211,6 +211,45 @@ def test_validate_if_ssh_params_are_added(self):
             "secret": {"secretName": "ssh-secret", "defaultMode": 288},
         } in jmespath.search("spec.template.spec.volumes", docs[0])
 
+    def test_validate_if_ssh_params_are_added_with_git_ssh_key(self):
+        docs = render_chart(
+            values={
+                "dags": {
+                    "gitSync": {
+                        "enabled": True,
+                        "containerName": "git-sync-test",
+                        "gitSshKey": "dummy-ssh-key",
+                        "knownHosts": None,
+                        "branch": "test-branch",
```

Comment:
We should either remove this or test it was set properly.

Reply:
removed

Reply:
It's still here.

Reply:
right, now removed in all the tests I added.